app/src/CoursePages/CoursePage.tsx

```diff
@@ -67,7 +68,12 @@ export const CourseInfo: React.FC<CourseInfoProps> = (props) => {
 
 
     const rateButtonClick = () => {
-        navigate('/course/' + props.classNum + '/review')
+        // User is not logged in
+        if (!getSignInStatus()) {
+            navigate('/login');
```

Comment:
This can be another feature in a different PR if we have the time to get to it, but it's a little bit jarring to click 'Rate' and suddenly be taken to the Login page without explanation. A sort of 'Log in to leave a review" notifier would be nice.